.github/workflows/build.yml

```diff
@@ -10,20 +10,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.11"]
```

Comment:
the python version that hatch uses to package impacts pip distribution & pypi/conda repositories. I don't think it's an issue to remove 3.8 builder however, 3.11 should be fine

Reply:
I've removed the matrix from Github Actions because, Hatch will take care of it:
```toml
[[tool.hatch.envs.test.matrix]]
# Python 3.7 does not works with pip-tools
python = ["3.8", "3.9", "3.10", "3.11", "3.12"]
```